src/components/section/section.css

```diff
@@ -0,0 +1,19 @@
+@import '../../styles/spacing.css';
+
+.description {
+  margin: 0;
+  margin-bottom: var(--spacing-large);
+}
+
+.section {
+  margin-bottom: var(--spacing-x-large);
+}
+
+.section > * {
```

Comment:
This rule is being applied to `subheading` as well! I'm also interested in seeing how this applies to a Section with many rows of children.

Reply:
Oooooh, good catch! I'll make sure it doesn't apply to subheading, even though the rule is identical. Better for semantics, I think.

My goal was to support `n` rows of children with this wildcard selector, and this way the rows don't have to be a particular type of element.

Reply:
On attempting to fix this, I had some more realizations:
- Both the subheading and description have the same `margin-bottom` as any other child element (like row containers).
- Any selector more precise would need to account for the description being optional (IE `*:nth-child(n+3)` would only work with a description present).
- Keeping the classes for a margin reset and moving down the section child selector dries up the css.

All that in mind, I opted to allow the section child selector to apply the `margin-bottom` rules. Lemme know if you have another preferred approach.

Reply:
I think `*` is fine since we are using CSS modules _and_ we are only affecting direct children (`>`).